pythran/toolchain.py

```diff
@@ -311,7 +311,13 @@ def compile_pythrancode(module_name, pythrancode, specs=None,
     # Autodetect the Pythran spec if not given as parameter
     from pythran.spec import spec_parser
     if specs is None:
-        specs = spec_parser(pythrancode)
+        specs = spec_parser(path_or_text)
+
+    if os.path.isfile(path_or_text):
+        with open(path_or_text) as fd:
+            pythrancode = fd.read()
+    else:
+        pythrancode = path_or_text
```

Comment:
BTW, shouldn't we replace non #pythran line by empty line in order to avoid wrong line information?